modules/test/playwright/tests/account-admin-web/accountGroups.spec.ts

```diff
@@ -16,6 +16,12 @@ export const test = mergeTests(
 	loginTest()
 );
 
+async function checkAccountGroupVisibility(page, accountGroup) {
+	await expect(
+		page.getByText(accountGroup.name, {exact: true})
+	).toBeVisible();
+}
+
```

Comment:
no, this should not go here: it should go inside the accountsPage but as a "dynamic locator". Have a look to `modules/test/playwright/pages/account-admin-web/EditAccountContactAddressPage.ts` for the addressDisplay